core/src/test/java/org/apache/calcite/test/MaterializedViewSubstitutionVisitorTest.java

```diff
@@ -1607,6 +1607,78 @@ private void checkSatisfiable(RexNode e, String s) {
             + "EnumerableTableScan(table=[[hr, MV0]])")).ok();
   }
 
+  @Test void testGroupSetLiteral() {
+    final String mv = ""
+        + "select \"deptno\", \"empid\"\n"
+        + "from \"emps\"\n"
+        + "group by \"deptno\", \"empid\"";
+    final String query = ""
+        + "select \"deptno\", 1\n"
+        + "from \"emps\"\n"
+        + "group by \"deptno\",  1";
+    sql(mv, query).ok();
+  }
+
+  @Test void testGroupSetMultipleLiteral() {
```

Comment:
Nitpick: `testGroupSetMultipleLiteral` -> `testGroupSetMultipleLiterals` (also in the other test names where literal is plural)

Reply:
thanks, update the code.

Reply:
With the latest commit I still see `testGroupSetMultipleLiteral`, can you double check?

Reply:
ok,thanks.